data/rao-result/rao-result-impl/src/test/java/com/farao_community/farao/data/rao_result_impl/RaoResultImplTest.java

```diff
@@ -205,4 +207,15 @@ public void testCostResults() {
 
         assertEquals(ComputationStatus.DEFAULT, raoResult.getComputationStatus());
     }
+
+    @Test
+    public void testRaoWentThroughSecondPrev() {
```

Comment:
i don't understand the name of the test : FIRST_PREV... doesn't mean that rao went through 2P, does it ?